app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -123,6 +129,24 @@ def process_collection(collection, key)
         end
       end
 
+      def gather_data_for_this_region(arm_service)
+        results = []
+        @data[:resource_groups].each do |resource_group|
+          results << arm_service.list(resource_group[:name])
```

Comment:
I don't know of a method that gets all resources with detailed information, sorry.